application/controllers/SurveyAdministrationController.php

```diff
@@ -2006,6 +1983,7 @@ public function actionOrganize()
 
         $iSurveyID = $this->getSurveyIdFromGetRequest();
 
```

Comment:
Maybe you could combine those two lines to only one? `$iSurveyID = sanitize_int($this->getSurveyIdFromGetRequest));`
What do you think?

Reply:
Yes sure :). Thanks Jessi